Description: Drop gtk_set_locale() which is deprecated
Author: Stephen Kitt <steve@sk2.org>

gtk_init() does everything necessary anyway...

Index: gdmap/src/main.c
===================================================================
--- gdmap.orig/src/main.c	2013-06-22 17:19:57.688844857 +0200
+++ gdmap/src/main.c	2013-06-22 17:21:43.915514364 +0200
@@ -60,7 +60,6 @@
   }
   g_option_context_free(context);
   
-  gtk_set_locale();
   gtk_init(&argc, &argv);
   
   gtk_rc_parse_string(GtkResource);
